worlds/ahit/test/TestActs.py

```diff
@@ -0,0 +1,35 @@
+from worlds.ahit.Regions import act_chapters
+from worlds.ahit.Rules import act_connections
+from . import HatInTimeTestBase
+
+
+class TestActs(HatInTimeTestBase):
+    def run_default_tests(self) -> bool:
+        return False
+      
+    def testAllStateCanReachEverything(self):
```

Comment:
?

Reply:
I was trying to disable that test in particular and didn't realize that run_default_tests had to be set to false instead of being overidden.